src/components/common/Headline/Headline.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Headline() {
+    return {
+        <h1 type="post_title"/>
```

Comment:
what is an attribute `type`?

Reply:
now I wrote this code <h1 className={post_title}/>

Reply:
yes